string map_name
---
bool is_success
string status